docs/whats-new.rst

```diff
@@ -51,6 +51,9 @@ Bug Fixes
   writing to netCDF files to avoid bugs when using ``libnetcdf``
   version 4.5.0 (:pull:`235`).  By `Spencer Hill
   <https://github.com/spencerahill>`_.
+- Skip ``Calc`` object calculation that computes time average of
```

Comment:
Expand a bit: note that this refers to when using `CalcSuite` (and thus `submit_mult_calcs`), and that `Calc` now raises a ValueError when instantiated with a non-time-defined variable but with one or more time-defined reductions.